android_lib/src/main/java/io/sentry/godotplugin/SentryAndroidGodotPlugin.kt

```diff
@@ -0,0 +1,394 @@
+    }
+
+    @UsedByGodot
+    fun getLastEventId(): String {
+        return Sentry.getLastEventId().toString()
+    }
+
+    @UsedByGodot
+    fun createEvent(): Int {
+        val event = SentryEvent()
+        val handle = registerEvent(event)
+        return handle
+    }
+
+    @UsedByGodot
+    fun releaseEvent(eventHandle: Int) {
+        val eventsMap = eventsByHandle.get() ?: run {
+            Log.e(TAG, "Internal Error -- eventsByHandle is null")
+            return
+        }
+
+        eventsMap.remove(eventHandle)
+    }
+
+    @UsedByGodot
+    fun captureEvent(eventHandle: Int): String {
+        val event: SentryEvent = getEvent(eventHandle) ?: run {
+            Log.e(TAG, "Failed to capture event: $eventHandle")
+            return ""
+        }
+        val id = Sentry.captureEvent(event)
+        return id.toString()
+    }
+
+    @UsedByGodot
+    fun eventGetId(eventHandle: Int): String {
+        val id = getEvent(eventHandle)?.eventId ?: return ""
+        return id.toString()
+    }
+
+    @UsedByGodot
+    fun eventSetMessage(eventHandle: Int, message: String) {
+        val event: SentryEvent = getEvent(eventHandle) ?: return
+        val sentryMessage = Message()
+        sentryMessage.formatted = message
+        event.message = sentryMessage
+    }
+
+    @UsedByGodot
+    fun eventGetMessage(eventHandle: Int): String {
+        return getEvent(eventHandle)?.message?.formatted ?: return ""
+    }
+
+    @UsedByGodot
+    fun eventSetTimestamp(eventHandle: Int, timestamp: String) {
+        val event: SentryEvent = getEvent(eventHandle) ?: return
+        try {
+            event.timestamp = timestamp.parseTimestamp()
+        } catch (_: Exception) {
+            Log.e(TAG, "Failed to parse timestamp: $timestamp")
+        }
+    }
+
+    @UsedByGodot
+    fun eventGetTimestamp(eventHandle: Int): String {
+        val event: SentryEvent = getEvent(eventHandle) ?: return ""
+        return event.timestamp?.toRfc3339() ?: ""
+    }
+
+    @UsedByGodot
+    fun eventGetPlatform(eventHandle: Int): String {
+        return getEvent(eventHandle)?.platform ?: return ""
+    }
+
+    @UsedByGodot
+    fun eventSetLevel(eventHandle: Int, level: Int) {
+        getEvent(eventHandle)?.level = level.toSentryLevel()
+    }
+
+    @UsedByGodot
+    fun eventGetLevel(eventHandle: Int): Int {
+        return getEvent(eventHandle)?.level?.toInt() ?: return SentryLevel.ERROR.toInt()
+    }
+
+    @UsedByGodot
+    fun eventSetLogger(eventHandle: Int, logger: String) {
+        getEvent(eventHandle)?.logger = logger
+    }
+
+    @UsedByGodot
+    fun eventGetLogger(eventHandle: Int): String {
+        return getEvent(eventHandle)?.logger ?: return ""
+    }
+
+    @UsedByGodot
+    fun eventSetRelease(eventHandle: Int, release: String) {
+        getEvent(eventHandle)?.release = release
+    }
+
+    @UsedByGodot
+    fun eventGetRelease(eventHandle: Int): String {
+        return getEvent(eventHandle)?.release ?: ""
```

Comment:
can we return `null` instead? Not sure about returning a value `""` all over, it'll show in Sentry as having the key and the vallue `""` where in fact we should avoid setting the key to begin with, if there's no value